identity/src/main/java/com/stripe/android/identity/ui/IndividualScreen.kt

```diff
@@ -0,0 +1,224 @@
+                    .weight(1f)
+                    .verticalScroll(rememberScrollState())
+            ) {
+                Text(
+                    text = individualPage.title,
+                    fontSize = 24.sp,
+                    fontWeight = FontWeight.Bold,
+                    modifier = Modifier.testTag(INDIVIDUAL_TITLE_TAG)
+                )
+                if (missing.contains(Requirement.NAME)) {
+                    NameSection {
+                        collectedName = it
+                        submitButtonState = updateSubmitButtonState(
+                            listOf(
+                                collectedDob,
+                                collectedAddress,
+                                collectedName,
+                                collectedIdNumber
+                            )
+                        )
+                    }
+                }
+                if (missing.contains(Requirement.DOB)) {
+                    DOBSection {
+                        collectedDob = it
+                        submitButtonState = updateSubmitButtonState(
+                            listOf(
+                                collectedDob,
+                                collectedAddress,
+                                collectedName,
+                                collectedIdNumber
+                            )
+                        )
+                    }
+                }
+                if (missing.contains(Requirement.IDNUMBER)) {
+                    IDNumberSection(
+                        individualPage.idNumberCountries,
+                        individualPage.idNumberCountryNotListedTextButtonText,
+                        isStandalone,
+                        navController
+                    ) {
+                        collectedIdNumber = it
+
+                        submitButtonState = updateSubmitButtonState(
+                            listOf(
+                                collectedDob,
+                                collectedAddress,
+                                collectedName,
+                                collectedIdNumber
+                            )
+                        )
+                    }
+                }
+                if (missing.contains(Requirement.ADDRESS)) {
+                    AddressSection(
+                        identityViewModel,
+                        individualPage.addressCountries,
+                        individualPage.addressCountryNotListedTextButtonText,
+                        isStandalone,
+                        navController
+                    ) {
+                        collectedAddress = it
+                        submitButtonState = updateSubmitButtonState(
+                            listOf(
+                                collectedDob,
+                                collectedAddress,
+                                collectedName,
+                                collectedIdNumber
```

Comment:
these 4 are always together, maybe it makes sense to create a Form wrapper class for them?

Reply:
wrapped in `IndividualCollectedStates`